src/main/java/com/conveyal/r5/analyst/Grid.java

```diff
@@ -516,7 +575,7 @@ public static Polygon getPixelGeometry (int x, int y, int zoom) {
 
         int i = 0;
         while (reader.readRecord()) {
-            if (++i % 10000 == 0) {
+            if (++i % 1000 == 0) {
```

Comment:
This will generate a lot of output, probably not useful outside development environment.